changelog.md

```diff
@@ -1,5 +1,34 @@
 # cxxeffect
 
+2021-08-02 update:
+
+I've reorganized the project, with library files going in the include/ directory,
+and example programs going into the examples directory.
+The cat program now looks like:
+
+```cpp
+#include <cxxeffect/stream.hpp>
+#include <cxxeffect/task.hpp>
+
+int main() {
+    eff::stream<eff::task, char> in = eff::io::cin<eff::task>(1024);
+    eff::pipe<eff::task, char, eff::bot> out = eff::io::cout<eff::task>();
+
+    eff::stream<eff::task, eff::bot> program = in.through(out);
+
+    eff::task<eff::top> result = program.compile().drain();
+
+    result.unsafeRunAsync().get();
+}
+```
+
+The project can be compiled with:
+```bash
+g++ examples/cat.cpp -Iinclude -pthread -std=c++17
+```
+
+Or by running `bash build.sh`, which will invoke CMake.
+
 2021-04-30 update:
 
 I don't think I'm going to work on this for the forseeable future. I don't have
```

Comment:
Now that this is collaborative, it might be good to get rid of these sorts of updates and make a real README

Reply:
Ah, I didn't see you moved this file.